en/concepts/scene/light/pbr-lighting.md

```diff
@@ -1,60 +1,58 @@
-# Physically-based Lighting
+# Physically Based Lighting
```

Comment:
```suggestion
# Physically-based Lighting
```

Reply:
According to Grammerly, this should contain a dash.  Change all instances?

Reply:
No, in Wiki there's no dash. See: https://en.wikipedia.org/wiki/Physically_based_rendering

Reply:
And here: https://learn.unity.com/tutorial/creating-physically-based-materials-unity-2019-3

Reply:
thank you, I'll report this as incorrect to Grammarly.